src/client/pages/Offer/Checkout/Sign.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
```

Comment:
Yes, we refactored it to avoid using graphql types as props.

We also got rid of the extra `SignStatus` component.

And we moved `SignUiState` to the parent component.